### ISSUE TITLE
about training 

### ISSUE THREAD

**Issue by suke27** [jmiller656/EDSR-Tensorflow#37]
Hello, I found you did not do normalization for input data, just do reduce mean op.
training can work well even without normalization ?

**Comment by jmiller656** [comment]
Empirically, it can!

I can't give any answers as to why this is done though, I'm just following the paper and am not one of the authors. So I can't explain why they don't do normalization other than subtracting the mean.
